recipe/meta.yaml

```diff
@@ -32,7 +32,8 @@ requirements:
     - patch       # [not win]
     - m2-patch    # [win]
   host:
-    - gflags
+    - gflags 2.2
+    - gtest
```

Comment:
You'll need to add it to `ignore_run_exports` unless you want ti to be in the runtime dependencies. @cbouss why did you request gtest to be added?

Reply:
My comment is still unaddressed. As far as I see, we don't run the tests so gtest is not needed.

Reply:
oh right, it's only used if `BUILD_TESTING=ON`
In that case @M-Waszkiewicz-Anaconda can you remove gtest? But then bld.bat should set testing off:
https://github.com/AnacondaRecipes/glog-feedstock/blob/PKG-3988/recipe/bld.bat#L8